mocks/landscape/landscaperepl/app.go

```diff
@@ -76,6 +78,31 @@ on your command line. Hosted at the specified address.`,
 			}()
 			defer server.Stop()
 
+			fileserverDir := cmd.Flag("fileserver-dir").Value.String()
```

Comment:
This is a very good idea and will save up some cycles. I wonder if we should though not just serve cwd when starting and not messing with an option for simplicity, wdyt?

Reply:
I consider the command line flag simple enough.

The alternative requires either one remembering to `cd` first and then run the landscaperepl from that location, which requires invoking if with its absolute path or store the intended files to be served alongside the code.

Finally. with the command line flag we can avoid serving files if we so desire.

Reply:
Right, but if 90% of the cases of usage requires the flag, then, it’s a little bit of a pity to have to request that.

Also, what happens in the mock if you use the API to request a rootfs but didn’t use the flag?

Reply:
Nothing would happen to the mock, but the agent would fail to download since we'd send URLs pointing to nowhere.
I'm removing the CLI based on the argument that 90% of the cases the REPL will run requires a file server.